test/fixture/snapshots/test-sourcemaps/build/feature/__tests__/test.js

```diff
@@ -0,0 +1,9 @@
+import test from '../../../../../../..';
+test('feature test title', t => {
+    t.snapshot({ foo: 'bar' });
+    t.snapshot({ answer: 40 });
+});
+test('another feature test', t => {
+    t.snapshot(new Map());
+});
+//# sourceMappingURL=test.js.map
```

Comment:
Is that just for constency? in this case everything under `...build/* ` is directly from the typescript compiler, I usually hesitate to edit compiled files for style only.